schemas/tslint/schema.json

```diff
@@ -0,0 +1,5832 @@
+                    },
+                    {
+                      "$ref": "#/definitions/rules/properties/no-empty/definitions/options/items"
+                    }
+                  ]
+                },
+                "severity": {}
+              },
+              "additionalProperties": false
+            }
+          ]
+        },
+        "no-eval": {
+          "description": "Disallows `eval` function invocations.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/rule"
+            },
+            {
+              "items": [
+                {
+                  "type": "boolean"
+                }
+              ],
+              "additionalItems": false,
+              "properties": {
+                "severity": {}
+              },
+              "additionalProperties": false
+            }
+          ]
+        },
+        "no-for-in": {
+          "description": "Forbid for…in statements\nhttps://palantir.github.io/tslint/rules/no-for-in/",
+          "allOf": [
+            {
+              "$ref": "#/definitions/rule"
+            },
+            {
+              "items": [
+                {
+                  "type": "boolean"
+                }
+              ],
+              "additionalItems": false,
+              "additionalProperties": false
+            }
+          ]
+        },
+        "no-for-in-array": {
+          "description": "Disallows iterating over an array with a for-in loop.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/rule"
+            },
+            {
+              "items": [
+                {
+                  "type": "boolean"
+                }
+              ],
+              "additionalItems": false,
+              "properties": {
+                "severity": {}
+              },
+              "additionalProperties": false
+            }
+          ]
+        },
+        "no-implicit-dependencies": {
+          "description": "Disallows importing modules that are not listed as dependency in the project's package.json.",
+          "definitions": {
+            "options": {
+              "type": "array",
+              "items": [
+                {
+                  "type": "string",
+                  "enum": [
+                    "dev",
+                    "optional"
+                  ]
+                },
+                {
+                  "type": "array"
+                }
+              ],
+              "minItems": 0,
+              "maxItems": 3
+            }
+          },
+          "allOf": [
+            {
+              "$ref": "#/definitions/rule"
+            },
+            {
+              "items": [
+                {
+                  "type": "boolean"
+                }
+              ],
+              "maxItems": 3,
+              "additionalItems": {
+                "$ref": "#/definitions/rules/properties/no-implicit-dependencies/definitions/options/items"
```

Comment:
@michaelmior This reference, for example, is invalid, as it options to an array that is by definition not a valid schema